agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/configuration/ConfigurationBuilder.java

```diff
@@ -61,27 +67,66 @@ public static Configuration create(Path agentJarPath) throws IOException {
         Configuration config = loadConfigurationFile(agentJarPath);
         PreviewConfiguration preview = config.instrumentationSettings.preview;
 
-        preview.roleName = overlayWithEnvVar(APPLICATIONINSIGHTS_ROLE_NAME, WEBSITE_SITE_NAME, preview.roleName);
+        preview.roleName = overlayWithEnvVars(APPLICATIONINSIGHTS_ROLE_NAME, WEBSITE_SITE_NAME, preview.roleName);
         preview.roleInstance =
-                overlayWithEnvVar(APPLICATIONINSIGHTS_ROLE_INSTANCE, WEBSITE_INSTANCE_ID, preview.roleInstance);
+                overlayWithEnvVars(APPLICATIONINSIGHTS_ROLE_INSTANCE, WEBSITE_INSTANCE_ID, preview.roleInstance);
 
+        loadSamplingRateEnvVar(preview);
+        loadLogCaptureEnvVar(preview);
```

Comment:
i think we could do the same here, by initializing `preview.instrumentation` map in PreviewConfiguration constructor, e.g.

```
        public PreviewConfiguration() {
            instrumentation.put("logging", new HashMap<>());
        }
```

then this could be simpler also:
```suggestion
        Map<String, Object> logging = preview.instrumentation.get("logging");
        logging.put("threshold", overlayWithEnvVar(APPLICATIONINSIGHTS_LOG_CAPTURE_THRESHOLD, logging.get("threshold")));
```